Basic-Video-Transformer/README.md

```diff
@@ -8,23 +8,94 @@ video APIs. It is very similar to the [Basic Video Chat](../Basic%20Video%20Chat
 
 You can see a demo of this sample running at [opentok.github.io/opentok-web-samples/Basic-Video-Transformer/](https://opentok.github.io/opentok-web-samples/Basic-Video-Transformer/)
 
-> **Note** The demo is setup so that a new room is generated based on your public IP address. So will only work if you are connecting from 2 browsers on the same network.
+> **Note** The demo is set up so that a new room is generated based on your public IP address. So it will only work if you are connecting from two browsers on the same network.
+
+## Installing dependencies
+
+Before running the app, install the Node modules it uses:
+
+```
+npm install
+```
 
 ## Running the App
 
 *Important:* Read the following sections of the main README file for the repository to set up
 and test the application:
 
 * [Setting up the test web service](../README.md#setting-up-the-test-web-service)
+
 * [Configuring the application](../README.md#configuring-the-application)
-* [Testing the application](../README.md#testing-the-application)
 
-## Transforming the Video Stream
+* [Testing the application](../README.md#testing-the-application) -- Note however that this
+  sample requires that you serve the index.html page via an http://localhost server or an
+  HTTPS server. This is because the sample loads the app.js script with `type` set to `"module"`:
+  
+  ```
+  <script type="module" src="js/app.js"></script>
+  ```
+
+  Modules are not supported in pages loaded with a file: URL scheme.
```

Comment:
Do we also need to mention that you need to run this sample in an incognito window? And is that true just for this sample? Can we say why?

Reply:
I assume that the incognito window is only required when testing locally.

Reply:
Hi Jeff,
We tried reproducing this a few times between Misha's computer and my computer and we couldn't see the error again. Do you still think we include this?